src/App.jsx

```diff
@@ -14,70 +15,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {
+        good
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${good} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  setGood('');
+                }}
+              />
+            </h1>
+          )
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              No goods selected
+            </h1>
+          )
+      }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {
+            goods.map((item) => {
+              const selectedGood = item === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              return (
+                <tr
+                  data-cy="Good"
+                  className={
+                    selectedGood ? 'has-background-success-light' : ''
```

Comment:
It's better to use classname library for making a dynamic classname

Reply:
Maybe, but it's not necessary in such a small project. In bigger projects, I'd better use some kind of 'CSS in JS' library.
**Fix the task description and include all conditions.**

Reply:
Actually it's not a good practice
So, please use this library

Reply:
It's not a good practice to use external libraries for just one line of code. What is the benefit of using the library in this case? It won't improve readability, but it certainly will increase the bundle size.

The reasons I would not use it are:

Unnecessary use of resources: Libraries take up a significant amount of space and require more time to install and load, which is impractical for one-off tasks.

Difficulty in managing dependencies: Installing large libraries can lead to dependency conflicts in your project, especially if they contain overlapping or deprecated components.

Performance and runtime: Libraries can slow down the load and runtime of your project, especially if they load many unused modules.

Maintainability: Fewer dependencies make it easier to maintain and update your project.